### ISSUE TITLE
API push

### ISSUE THREAD

**Issue by betz** [audreyt/ethercalc#558]
Hi,

I would like to have ethercalc updates pushed to a specific URL.
If a call is updated, ethercalc calls a url with the changes.
Is this possible?

Cheers!
Tom

**Comment by eddyparkinson** [comment]
Do you mean when ethercalc is updated on git hub?  - There are tools that do this, have not looked at them.

Or are you thinking of the ethercalc server pushing spreadsheet data? - It has a way to send email, but not trigger a URL

**Comment by betz** [comment]
No, i am looking in ethercalc pushing spreadsheet data. But in realtime.
On cell, or at least row level.

And is there a way i could hack it in? Where to start looking if I may ask?

**Comment by eddyparkinson** [comment]
Are you pushing to a browser or to a server. Ethercalc already pushes every change to each connected browser that is using the sheet.  But if you want to push to a server via a url, then you would need some extra code.   
I expect here:
https://github.com/audreyt/ethercalc/blob/master/src/sc.ls#L132
This server code takes spreadsheet commands and passes them to the servers copy of the spreadsheet. This keeps the server copy of the spreadsheet in sync with the clients.
